app/assets/stylesheets/rad_common/rad_common.scss

```diff
@@ -210,6 +210,12 @@ td.fa.security-group-td { border-style: none; }
   }
 }
 
+.form-inline {
```

Comment:
Can we limit this to a particular ID in addition to the class? It looks so generic that it could be used elsewhere.

Reply:
agreed, I was going for something more generic at first, but this is becoming more specific so adding a custom class